lib/index.js

```diff
@@ -372,6 +404,32 @@ function opt (option) {
   }
 };
 
+/* chan and rmchan sockopt methods. only relevant for subscription sockets */
+Socket.prototype.chan       = function(list){
```

Comment:
oof, i think i pushed the wrong state. haha it's on my home computer. anyway, `this.register` function iterates the list. i'll update that to reflect list as an array as discussed earlier